addons/delivery/static/src/js/location_selector_field/pickup_location_field.js

```diff
@@ -0,0 +1,75 @@
+import { buildM2OFieldDescription, extractM2OFieldProps, Many2OneField } from "@web/views/fields/many2one/many2one_field";
```

Comment:
Could you please change to single quotes everywhere? we use single quotes for technical strings on our scope.
```suggestion
import { buildM2OFieldDescription, extractM2OFieldProps, Many2OneField } from '@web/views/fields/many2one/many2one_field';
```